services/metadata/backend/data/bruno-manser.json

```diff
@@ -90,8 +90,8 @@
         "url": {
             "__type": "URL",
             "type": "URL",
-            "url": "https://data.dasch.swiss/bmf/",
-            "text": "https://data.dasch.swiss/bmf/"
+            "url":
+            "text":
```

Comment:
this is not valid JSON syntax and I'm concerned it might break the server to try to upload this.